helm-charts/Chart.yaml

```
@@ -21,3 +21,18 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 appVersion: 6.0.1
+
+# Dependencies
+dependencies:
+  - name: redis 
+    version: 10.5.6
+    repository: "https://kubernetes-charts.storage.googleapis.com/"
+    condition: redis.enabled
+  - name: mongodb
+    version: 8.2.1
+    repository: "https://charts.bitnami.com/bitnami"
+    condition: mongodb.enabled
+  - name: postgresql
+    version: "8.6.4"
```

Are the quotes necessary?

```suggestion
    version: 8.6.4
```